src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -953,6 +970,26 @@ export default class GeFormRenderer extends NavigationMixin(LightningElement) {
         return this.dataImport && this.dataImport[STATUS_FIELD.fieldApiName] === 'Imported';
     }
 
+    /**
+     * Combine fabricated names and lookup names, giving priority to non-empty fabricated names
+     */
+    @api
+    getCardholderNames() {
+        const names = this.fabricatedCardholderNames;
+        const firstName = isNotEmpty(names.firstName) ? names.firstName : this._contact1FirstName;
+        const lastName = isNotEmpty(names.lastName) ? names.lastName : this._contact1LastName;
+        const accountName = isNotEmpty(names.lastName) ? names.lastName : this._contact1LastName;
```

Comment:
Looks like we need to set this equal to `names.accountName` or maybe just use `names.accountName` directly when the donor is an account?